tests/testthat/test-generalizedlinearmodel.R

```diff
@@ -0,0 +1,314 @@
+# jaspTools::testAnalysis("GeneralizedLinearModel")
+
+#library(testthat)
+context("Classical GLM Unit Testing")
```

Comment:
```suggestion
context("GeneralizedLinearModel")
```
It's nice to return just the name of the main function